src/sentry/models/groupowner.py

```diff
@@ -15,6 +15,12 @@ class GroupOwnerType(Enum):
     OWNERSHIP_RULE = 1
 
 
+GROUP_OWNER_TYPE = {
+    GroupOwnerType.SUSPECT_COMMIT: "Suspect Commit",
+    GroupOwnerType.OWNERSHIP_RULE: "Ownership Rule",
```

Comment:
So I'm sending "Suspect Commit"/"Ownership Rule". Not sure if the frontend would rather something like "suspect_commit" / "ownership_rule"?

Reply:
should be fine on the frontend either way, not sure if there is a standard tho for snakecase :/

Reply:
Many of the enums we have in the UI today use `camelBack` form. So `suspectCommit` and `ownershipRule` would work.

Reply:
Changing to that 👍